discord/embeds.py

```diff
@@ -374,7 +374,7 @@ def __init__(
         if timestamp:
             self.timestamp = timestamp
 
-        self._fields: list[EmbedField] = fields
+        self._fields: list[EmbedField] = fields if fields is not None else []
```

Comment:
Simplify to previous version
```suggestion
        self._fields: list[EmbedField] = fields or []
```

Reply:
Checking for None is not the same as checking for falsey values
This is also more explicit, and easier to follow, so it's better in the original